data/special_routes.yaml

```
@@ -234,3 +234,9 @@
   :additional_routes:
     - :base_path: "/when-do-the-clocks-change.json"
       :type: "exact"
+
+- :content_id: '2b4e10dd-26eb-4e67-bf0e-1c9f56e60388'
+  :base_path: '/apply-for-a-licence'
+  :type: "prefix"
+  :title: 'Apply for a licence'
```

Does it need a description? 🤔 

```suggestion
  :title: 'Apply for a licence'
  :description: 'Find licences, permits or certifications you may need for yourself or your business'
```

---

From the readme its appears not to be a require field. Because this special route is solely used for routing the title and description are never used.